packages/tables/src/Actions/CreateAction.php

```diff
@@ -58,16 +58,13 @@ protected function setUp(): void
                     $data = Arr::except($data, $pivotColumns);
                 }
 
-                $record = $relationship->create($data);
-
+                $pivotData = [];
                 if ($relationship instanceof BelongsToMany) {
                     $pivotData = Arr::only($data, $pivotColumns);
-
-                    if (count($pivotColumns)) {
-                        $record->{$relationship->getPivotAccessor()}->update($pivotData);
-                    }
                 }
 
+                $record = $relationship->create($data, $pivotData);
```

Comment:
The pivot data should still only be added if the `$relationship instanceof BelongsToMany`, right?

Reply:
It is fixed. Thanks for the swift review.